2.14/2.14.4/index.html

```diff
@@ -0,0 +1,268 @@
+			break;
+		case 21:
+			sum = 'двадцать один';
+			break;
+		case 22:
+			sum = 'двадцать два';
+			break;
+		case 23:
+			sum = 'двадцать три';
+			break;
+		case 24:
+			sum = 'двадцать четыре';
+			break;
+		case 25:
+			sum = 'двадцать пять';
+			break;
+		case 26:
+			sum = 'двадцать шесть';
+			break;
+		case 27:
+			sum = 'двадцать семь';
+			break;
+		case 28:
+			sum = 'двадцать восемь';
+			break;
+		case 29:
+			sum = 'двадцать девять';
+			break;
+		case 30:
+			sum = 'тридцать';
+			break;
+		case 31:
+			sum = 'тридцать один';
+			break;
+		case 32:
+			sum = 'тридцать два';
+			break;
+		case 33:
+			sum = 'тридцать три';
+			break;
+		case 34:
+			sum = 'тридцать четыре';
+			break;
+		case 35:
+			sum = 'тридцать пять';
+			break;
+		case 36:
+			sum = 'тридцать шесть';
+			break;
+		case 37:
+			sum = 'тридцать семь';
+			break;
+		case 38:
+			sum = 'тридцать восемь';
+			break;
+		case 39:
+			sum = 'тридцать девять';
+			break;
+		case 40:
+			sum = 'сорок';
+			break;
+		case 41:
+			sum = 'сорок один';
+			break;
+		case 42:
+			sum = 'сорок два';
+			break;
+		case 43:
+			sum = 'сорок три';
+			break;
+		case 44:
+			sum = 'сорок четыре';
+			break;
+		case 45:
+			sum = 'сорок пять';
+			break;
+		case 46:
+			sum = 'сорок шесть';
+			break;
+		case 47:
+			sum = 'сорок семь';
+			break;
+		case 48:
+			sum = 'сорок восемь';
+			break;
+		case 49:
+			sum = 'сорок девять';
+			break;
+		case 50:
+			sum = 'пятьдесят';
+			break;
+		case 51:
+			sum = 'пятьдесят один';
+			break;
+		case 52:
+			sum = 'пятьдесят два';
+			break;
+		case 53:
+			sum = 'пятьдесят три';
+			break;
+		case 54:
+			sum = 'пятьдесят четыре';
+			break;
+		case 55:
+			sum = 'пятьдесят пять';
+			break;
+		case 56:
+			sum = 'пятьдесят шесть';
+			break;
+		case 57:
+			sum = 'пятьдесят семь';
+			break;
+		case 58:
+			sum = 'пятьдесят восемь';
+			break;
+		case 59:
+			sum = 'пятьдесят девять';
+			break;
+		case 60:
+			sum = 'шестьдесят';
+			break;
+		case 61:
+			sum = 'шестьдесят один';
+			break;
+		case 62:
+			sum = 'шестьдесят два';
+			break;
+		case 63:
+			sum = 'шестьдесят три';
+			break;
+		case 64:
+			sum = 'шестьдесят четыре';
+			break;
+		case 65:
+			sum = 'шестьдесят пять';
+			break;
+		case 66:
+			sum = 'шестьдесят шесть';
+			break;
+		case 67:
+			sum = 'шестьдесят семь';
+			break;
+		case 68:
+			sum = 'шестьдесят восемь';
+			break;
+		case 69:
+			sum = 'шестьдесят девять';
+			break;
+		case 70:
+			sum = 'семьдесят';
+			break;
+		case 71:
+			sum = 'семьдесят один';
+			break;
+		case 72:
+			sum = 'семьдесят два';
+			break;
+		case 73:
+			sum = 'семьдесят три';
+			break;
+		case 74:
+			sum = 'семьдесят четыре';
+			break;
+		case 75:
+			sum = 'семьдесят пять';
+			break;
+		case 76:
+			sum = 'семьдесят шесть';
+			break;
+		case 77:
+			sum = 'семьдесят семь';
+			break;
+		case 78:
+			sum = 'семьдесят восемь';
+			break;
+		case 79:
+			sum = 'семьдесят девять';
+			break;
+		case 80:
```

Comment:
tooooo loooong code ) it should be shorter
it's not a good approach because if we need to show numbers more than 1000, the code will be ten times larger